fog/test-client/src/counters.rs

```diff
@@ -85,6 +85,12 @@ lazy_static::lazy_static! {
     /// Number of times that the test failed because a balance check operation failed
     pub static ref CHECK_BALANCE_ERROR_COUNT: IntCounter = OP_COUNTERS.counter("check_balance_error_count");
 
+    /// Number of times that the test failed because a get fee operation failed
+    pub static ref GET_FEE_ERROR_COUNT: IntCounter = OP_COUNTERS.counter("get_fee_error_count");
```

Comment:
Thanks for adding this. This fee error was happening a lot during my Fog Local Network CI work, and it'll be nice to track this.